### ISSUE TITLE
XML node is destructive

### ISSUE THREAD

**Issue by JaZo** [n8n-io/n8n#2762]
**Describe the bug**
When using the JSON to XML node, the incoming data is changed. This does not follow the [general guidelines](https://docs.n8n.io/nodes/creating-nodes/general-guidelines.html#do-not-change-incoming-data).

**To Reproduce**
1. Execute the workflow
2. See that the logged item is edited and only contains the XML

Workflow:
```json
{
  "nodes": [
    {
      "parameters": {},
      "name": "Start",
      "type": "n8n-nodes-base.start",
      "typeVersion": 1,
      "position": [
        40,
        260
      ]
    },
    {
      "parameters": {
        "mode": "jsonToxml",
        "options": {}
      },
      "name": "Convert to XML",
      "type": "n8n-nodes-base.xml",
      "typeVersion": 1,
      "position": [
        440,
        260
      ],
      "notesInFlow": false
    },
    {
      "parameters": {
        "functionCode": "return [\n  {json: {\n    \"id\": \"1467e6ae-163f-4598-b70a-381c46649031\",\n    \"type\": \"orders\",\n    \"orderNumber\": \"2022-0045\",\n    \"orderDate\": \"2022-01-19T15:06:16\",\n  }}\n];"
      },
      "name": "Test data",
      "type": "n8n-nodes-base.function",
      "typeVersion": 1,
      "position": [
        240,
        260
      ]
    },
    {
      "parameters": {
        "functionCode": "console.log(item);\n\nreturn item;"
      },
      "name": "Log item",
      "type": "n8n-nodes-base.functionItem",
      "typeVersion": 1,
      "position": [
        440,
        460
      ]
    }
  ],
  "connections": {
    "Start": {
      "main": [
        [
          {
            "node": "Test data",
            "type": "main",
            "index": 0
          }
        ]
      ]
    },
    "Test data": {
      "main": [
        [
          {
            "node": "Convert to XML",
            "type": "main",
            "index": 0
          },
          {
            "node": "Log item",
            "type": "main",
            "index": 0
          }
        ]
      ]
    }
  }
}
```

**Expected behavior**
The JSON to XML node should clone the incoming data.

**Environment (please complete the following information):**
 - OS: Windows 10 Business (21H1)
 - n8n Version: 0.160.0
 - Node.js Version: 14.17.3
 - Database system: default
 - Operation mode: default

**Additional context**
It would also be nice if the JSON to XML node had some kind of a "Keep Only Set" option just like the Set node so it can be additive.

**Comment by janober** [comment]
Thanks a lot for making us aware of that. We will fix that asap.

**Comment by janober** [comment]
Got fixed and will be released with the next version.

**Comment by janober** [comment]
Got released with `n8n@0.162.0`
